watch-parties/src/main/webapp/style.css

```diff
@@ -0,0 +1,37 @@
+div{
```

Comment:
This will apply this style to every `div` on the page. Instead, this should just be included in the `#chat` rule below.